tests/simple/simple_test.go

```diff
@@ -113,30 +130,95 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestSimpleFile(t *testing.T) {
-	// Special case to handle test invocation without args. See TestMain()
-	// early return comment.
-	if rc == nil {
-		return
-	}
-	// Run the flag-configured tests.
-	for _, filename := range flag.Args() {
-		testFile, err := parseSimpleFile(filename)
-		if err != nil {
-			t.Fatalf("Can't parse input file %v: %v", filename, err)
+ func TestSimpleFile(t *testing.T) {
+        // Special case to handle test invocation without args. See TestMain()
+        // early return comment.
+        if rc == nil {
+                return
+        }
+	var skipTests []string
+	firstFlag := true
+	for _, flagVal := range skipFlags {
+		if !strings.Contains(flagVal, "/") {
+			log.Fatal("Unable to parse skip_test flag, specifically for ", flagVal)
```

Comment:
Make it more meaningful:  `skip_test argument must contain at least <file>/<section>:`.